main.tf

```diff
@@ -11,7 +11,7 @@ variable "fp_context" {
 }
 
 variable "commit_hash" {
-  type = string
+  type    = string
```

Comment:
Remove extra spaces here.

Reply:
This is Terraform style convention: https://www.terraform.io/docs/configuration/style.html

Running `terraform fmt` automatically formats Terraform code to their convention.

Reply:
Got you! Thank you for clarifying!